code/modules/projectiles/projectile/hookshot.dm

```diff
@@ -202,6 +202,15 @@
 		visible_message("<span class='warning'>[W.firer] whips it good!</span>")
 	..(atarget, blocked)
 
+/obj/item/projectile/hookshot/whip/liquorice
+	name = "Liquoricium Whip"
```

Comment:
is this a proper noun? the other one said it was custom so I figured it might be, but this one doesn't seem to be